### ISSUE TITLE
New repo based on yours....

### ISSUE THREAD

**Issue by bhuvidya** [webpatser/laravel-countries#80]
Hi Christophe. I needed a Laravel package to install country information and came across yours. I ended up doing a number of edits to it, in part because I wanted to go through the process of putting together a proper Laravel package, and publishing it on Packagist. There were too many edits to submit as pull requests.

So I ended up creating a fresh repo based on your latest codebase. I have credited you prominently in my repo - after all you did all the hard work of getting the info together!

Basically I'm telling you all this out of courtesy, and I hope you don't mind.

Cheers.
